src/components/list/ItemForm.vue

```diff
@@ -97,27 +102,40 @@ export default {
     closeItemForm() {
       this.$modal.hide('itemForm');
     },
-    clearData() {
-      this._setItemForEditting(null);
-      this.item = new models.Item();
-    },
     setItemForEditting() {
       if (this.edittingItemObj) {
         this.item = { ...this.edittingItemObj };
       }
     },
+    checkRequestStatus(event) {
+      if (this.isRequestProcessing) {
+        event.cancel();
+      }
```

Comment:
hmmm, let's discuss this thing - I'm not sure that it is a good idea cancel close event.